Tests fail to build on master`

I'm guessing since 94f5c03cb75bfe38d5e3e838d54af8eda1e14270 the tests fail to build.
```
/home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp: In member function ‘virtual void PlannerFunctions_testDirectionTree_Test::TestBody()’:
/home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp:617:66: error: invalid initialization of reference of type ‘avoidance::PolarPoint&’ from expression of type ‘Eigen::Vector3f {aka Eigen::Matrix<float, 3, 1>}’
   bool res = getDirectionFromTree(p, path_node_positions, postion);
                                                                  ^
In file included from /home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp:4:0:
/home/nico/catkin_ws/src/avoidance/local_planner/test/../src/nodes/planner_functions.h:73:6: note: in passing argument 1 of ‘bool avoidance::getDirectionFromTree(avoidance::PolarPoint&, const std::vector<geometry_msgs::Point_<std::allocator<void> > >&, const Vector3f&)’
 bool getDirectionFromTree(
      ^
/home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp:618:69: error: invalid initialization of reference of type ‘avoidance::PolarPoint&’ from expression of type ‘Eigen::Vector3f {aka Eigen::Matrix<float, 3, 1>}’
   bool res1 = getDirectionFromTree(p1, path_node_positions, postion1);
                                                                     ^
In file included from /home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp:4:0:
/home/nico/catkin_ws/src/avoidance/local_planner/test/../src/nodes/planner_functions.h:73:6: note: in passing argument 1 of ‘bool avoidance::getDirectionFromTree(avoidance::PolarPoint&, const std::vector<geometry_msgs::Point_<std::allocator<void> > >&, const Vector3f&)’
 bool getDirectionFromTree(
      ^
/home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp:619:69: error: invalid initialization of reference of type ‘avoidance::PolarPoint&’ from expression of type ‘Eigen::Vector3f {aka Eigen::Matrix<float, 3, 1>}’
   bool res2 = getDirectionFromTree(p2, path_node_positions, postion2);
                                                                     ^
In file included from /home/nico/catkin_ws/src/avoidance/local_planner/test/test_planner_functions.cpp:4:0:
/home/nico/catkin_ws/src/avoidance/local_planner/test/../src/nodes/planner_functions.h:73:6: note: in passing argument 1 of ‘bool avoidance::getDirectionFromTree(avoidance::PolarPoint&, const std::vector<geometry_msgs::Point_<std::allocator<void> > >&, const Vector3f&)’
 bool getDirectionFromTree(
      ^
CMakeFiles/local_planner-test.dir/build.make:158: recipe for target 'CMakeFiles/local_planner-test.dir/test/test_planner_functions.cpp.o' failed
```

Not sure why travis doesn't see this problem

--- a/local_planner/test/test_planner_functions.cpp
+++ b/local_planner/test/test_planner_functions.cpp
@@ -608,7 +608,7 @@ TEST(PlannerFunctions, testDirectionTree) {
   const std::vector<geometry_msgs::Point> path_node_positions = {n4, n3, n2, n1,
                                                                  n0};
 
-  Eigen::Vector3f p, p1, p2;
+  PolarPoint p, p1, p2;
   Eigen::Vector3f postion(0.2, 0.3, 1.5);
   Eigen::Vector3f postion1(1.1, 2.3, 2.5);
   Eigen::Vector3f postion2(5.4, 2.0, 2.5);
@@ -622,12 +622,12 @@ TEST(PlannerFunctions, testDirectionTree) {
   // node n3 and n4 for position1, and not to get an available tree for the
   // position2
   ASSERT_TRUE(res);
-  EXPECT_FLOAT_EQ(45.0f, p.x());
-  EXPECT_FLOAT_EQ(57.0f, p.y());
+  EXPECT_NEAR(45.0f, p.e, 1.f);
+  EXPECT_NEAR(57.0f, p.z, 1.f);
 
   ASSERT_TRUE(res1);
-  EXPECT_FLOAT_EQ(0.0f, p1.x());
-  EXPECT_FLOAT_EQ(72.0f, p1.y());
+  EXPECT_NEAR(0.0f, p1.e, 1.f);
+  EXPECT_NEAR(72.0f, p1.z, 1.f);
 
   ASSERT_FALSE(res2);
 }

